test/framework/management/kind/mgmt.go

```diff
@@ -266,36 +253,45 @@ func (c *Cluster) GetClient() (client.Client, error) {
 }
 
 // GetWorkloadClient returns a controller-runtime client for the workload cluster.
-// TODO Use remote package once it returns a controller runtime client
 func (c *Cluster) GetWorkloadClient(ctx context.Context, namespace, name string) (client.Client, error) {
```

Comment:
For future cleanup: We can use remote.NewClusterClient() if we move it to a util function.